Sources/xcodeproj/Objects/BuildPhase/PBXBuildFile.swift

```diff
@@ -10,7 +10,7 @@ public final class PBXBuildFile: PBXObject {
     /// Returns the file the build file refers to.
     public var file: PBXFileElement? {
         get {
-            return fileReference?.getObject()
+            return fileReference?.materialize()
```

Comment:
I've taken the opportunity to rename this method. I've taken the idea from other libraries that you the same verb to describe the action of unwrapping a wrapped value.